android-components/components/browser/state/src/main/java/mozilla/components/browser/state/state/BrowserState.kt

```diff
@@ -50,4 +50,5 @@ data class BrowserState(
     val restoreComplete: Boolean = false,
     val locale: Locale? = null,
     val showExtensionProcessDisabledPopup: Boolean = false,
+    val extensionsProcessDisabled: Boolean = false,
```

Comment:
If this keeps growing we may need to create a separate state (Class) to have all the extension process info.